web/src/ui/pages/home/Home.tsx

```diff
@@ -43,7 +44,25 @@ export default function Home(props: Props) {
     const statsCases = config.home.statistics.catgegories as Array<availableStat>;
 
     type availableUseCase = "declareReferent" | "editSoftware" | "addSoftwareOrService";
```

Comment:
majuscule pour les types , c'est les useCase de quoi ? ça mériterait peut-être d'apparaitre dans le nom :

```suggestion
    type AvailableUseCase = "declareReferent" | "editSoftware" | "addSoftwareOrService";
```

Reply:
Oui, j'ai vu qu'on avait laisser passé ça d'une ancienne PR :sweat_smile: